gui/src/gui_state_container.h

```diff
@@ -84,6 +81,8 @@ class GuiStateContainer : public QObject {
   Q_PROPERTY(qreal flowReadout READ get_flow_readout NOTIFY readouts_changed)
   Q_PROPERTY(qreal tvReadout READ get_tv_readout NOTIFY readouts_changed)
 
+  Q_PROPERTY(VentMode mode READ get_mode NOTIFY params_changed)
```

Comment:
This would be better done as one property of an enum type, but I can clean it up myself in a followup.

Reply:
Yeah, I tried, but I didn't have the enum type handy in the code that was doing the setting.

Anywho, thanks.